test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -70,19 +70,20 @@ declare module 'test-case-reporting' {
   /* eslint @typescript-eslint/camelcase: "off" */
   namespace DB {
     export interface Build {
-      // `id` is nullable because is not set when uploading, it is only set
-      // when we get the build from the database.
-      // It is not nullable in the database.
+      // Nullable because it is not set when uploading. Not nullable in the database.
```

Comment:
I'm not fully convinced by this rewording -- let me know if you want to change it back.

Reply:
I think it was fine before

Reply:
Removed.